vital/types/camera_rpc.h

```diff
@@ -143,13 +147,24 @@ class VITAL_EXPORT simple_camera_rpc :
    */
   simple_camera_rpc ( vector_3d &world_scale, vector_3d &world_offset,
                       vector_2d &image_scale, vector_2d &image_offset,
-                      rpc_matrix &rpc_coeffs ) :
+                      rpc_matrix &rpc_coeffs, unsigned int image_width=0,
+                      unsigned int image_height=0) :
     world_scale_( world_scale ),
     world_offset_( world_offset ),
     image_scale_( image_scale ),
     image_offset_( image_offset ),
-    rpc_coeffs_( rpc_coeffs )
+    rpc_coeffs_( rpc_coeffs ),
+    image_width_( image_width ),
+    image_height_( image_height )
   {
+    if (image_width_ == 0)
+    {
+      image_width_ = static_cast<unsigned int>(std::ceil(image_offset_[0]*2));
```

Comment:
See other comment on guessing image dimension in the perspective camera.  If we decide to make changes there, we should do the same here.